SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs

```diff
@@ -123,6 +126,38 @@ protected override string GetScreenSize()
             }
         }
 
+        private static string DeploymentVersion
+        {
+            get
+            {
+#if NET461
+                // Get ClickOnce version (does not exist on .NET Core). 
+                if (ApplicationDeployment.IsNetworkDeployed)
+                {
+                    return ApplicationDeployment.CurrentDeployment.CurrentVersion.ToString();
+                }
+#endif
+                return null;
+            }
+        }
+
+        private static string FileVersion
+        {
+            get
+            {
+                // The AssemblyFileVersion uniquely identifys a build.
+                var entryAssembly = Assembly.GetEntryAssembly();
+                if (entryAssembly != null)
+                {
+                    var fileVersionInfo = FileVersionInfo.GetVersionInfo(entryAssembly.Location);
+                    return fileVersionInfo.FileVersion;
+                }
+
+                // Fallback if entry assembly is not found (in unit tests for example).
+                return Application.ProductVersion;
```

Comment:
Shouldn't it be `DeploymentVersion ?? Application.ProductVersion` fallback?

Reply:
No, it's already a fallback if `DeploymentVersion` is not available